### ISSUE TITLE
Instance of an object from System.Windows.Input halves the point passed to Mouse.Click

### ISSUE THREAD

**Issue by mircea21S** [FlaUI/FlaUI#612]
**Describe the bug**
If I am trying to use `Mouse.Click(Point)` and in the same test I am having an unused instance of some object from System.Windows.Input namespace. The `Point` used by `Mouse.Click()` will have the coordinates halved.

**Code snippets**
`var x = new System.Windows.Input.MouseGesture();` 
`Mouse.Click(new Point(100,100))`

So the Point has X = 100, Y = 100 and on the screen the Mouse will be positioned at a Point that has X = 50, Y = 50. If I don't have any instance from System.Windows.Input it's positioning the mouse correctly.

**Comment by Roemer** [comment]
Do you have scaling / high-dpi active?

**Comment by mircea21S** [comment]
Yes, I do. Is there any way this could work without setting the scaling to 100%?
